### ISSUE TITLE
[UX] Date/Views: change the weird `field_*:value2` for end date to something less vague

### ISSUE THREAD

**Issue by klonos** [backdrop/backdrop-issues#3988]
Here's what comes up when a date field with both a start and an end date has been added to a content type:

![Screen Shot 2019-08-24 at 2 30 12 pm](https://user-images.githubusercontent.com/2423362/63636729-f8dcae80-c67b-11e9-8f44-b6d48ea59aa4.png)

That should be something like `field_*:end_date`

**Comment by indigoxela** [comment]
@klonos are you sure, this is a bug? This is exactly the same as in Drupal 7 with views.

"value2" is for instance part of the database column name for date fields. And date module always uses "value2" for end dates. So at least this is consistent.

"star date" and "end date" should make clear, what this field is about anyway.

**Comment by klonos** [comment]
OK, I guess what I am saying is that this `value2` is a "magical" name, and we cannot possibly expect novice Drupal/Backdrop users and devs to guess what it is wherever it may show up. Is there any use case where there is a `value3` and a `value4` ...and a `valuen`? If not, then why not call this thing what it is?

**Comment by klonos** [comment]
...and fair enough, this is not a bug per se (although we do treat UX/DX WTFs as bugs in Backdrop). So lets call it a task instead.

**Comment by indigoxela** [comment]
> I guess what I am saying is that this value2 is a "magical" name

I agree, this is somewhat magical. Is it worth changing it throughout all the date code, possibly including the database column names, though?

Or are you asking for a display change in views UI? Not sure, if this would be tricky...

**Comment by klonos** [comment]
> Is it worth changing it throughout all the date code...

The fact that there has been no resistance when this was initially implemented in this "magical" way, does not mean that we need to propagate this bad coding practice for eternity ...it's the same as naming variables `$var1`, `$var2`, or naming files `document`, `document_new`, `document_new_new` 🙄

I am not saying that this is top-priority; I usually open issues as to-dos, with the hope that eventually someone (this may be me) will come along, pick this up, and fix things. ...it may take years, and we may never get to it after all; but at least by raising such tickets, we show that we noticed, and that we care ❤️ ...lets do our best, so that future generations of Backdropers hate us less 😅
